conditional/util/ldap.py

```diff
@@ -117,6 +117,12 @@ def ldap_is_current_student(account):
     return _ldap_is_member_of_group(account, 'current_student')
 
 
+def ldap_get_housingpoints(account):
```

Comment:
I didn't think it was necessary to clear the cache here, but it might be a good idea, in case anything's been changed in LDAP directly. Thoughts?

Reply:
this seems really heavy handed for getting housing points for a specific member? since you aren't caching this function, I don't think caches even get interacted with?

Reply:
I figured from looking at other methods (any of the `ldap_set_*` methods) the cache was used by default. If that's not the case, I could remove it from the get method, but I also figure better safe than sorry

Reply:
Set I understand, since the caches that store members are likely invalidated, but get should be read only?